docs/en/reference/working-with-associations.rst

```diff
@@ -500,8 +500,27 @@ and the "remove" operation.
         //...
     }
 
-Even though automatic cascading is convenient it should be used
-with care. Do not blindly apply cascade=all to all associations as
+Since ``cascade: persist`` is configured for the ``User#commentsAuthored``
+association, you can now create a user and persist his comment as follows:
+
+.. code-block:: php
+
+    <?php
+    $user = new User();
+    $myFirstComment = new Comment();
+    $user->addComment($myFirstComment);
+    
+    $em->persist($user);
+    $em->flush();
+
+.. note::
+
+    This code does not associate the newly created comment with the user.
+    To achieve this, you *always* have to call ``$myFirstComment->setUser($user);`` –
+    no matter if ``cascade: persist`` is set or not.
```

Comment:
> Makes sense?

Yeah, **thanks**, it's starting to make sense :-)

IMO what you're saying should be explained in the docs! I think I'll somewhat rewrite the entire 8.6 chapter - right now the very first sentence is already leading to the wrong direction ("can become pretty cumbersome"), namely promoting the cascading as a *convenience* feature.

I would explain the question "who should use it" at first, deferring all other topics to later. Reason: For a feature which is only useful for a small minority, it doesn't make sense to start with details.

Is that OK?

However, one more question I have:  
What exactly do you mean by "expose to the outside world"? I mean, the Comment entity has to be instantiated anyway: `$myFirstComment = new Comment();` - so it is always "exposed" (in the sense of "visible", or "modifyable"), and never "hidden in the implementation".  
Or do you mean that it's not directly "persistable" by the PHP app - i.e. the only way to get a Comment into the database is "through" the User entity?